### DISCUSSION TITLE
Is there an image picker for web?

### DISCUSSION THREAD

**Discussion by VTSingle** [necolas/react-native-web#2224] [Help]
Hey guys. 
With what package can I get an image from the gallery and upload it as a user avatar?

In the mobile version of the application, I use - **react-native-image-picker**

**Comment by Dovakeidy** [comment]
There are apparently two libraries that are labeled as web compatible in reactnative.directory ( https://reactnative.directory/?web=true&search=image+picker ). The one you mentioned is apparently web compatible now (although not all options seem to be supported)
